### DISCUSSION TITLE
Theme.json To Set MinHeight of Core/Cover

### DISCUSSION THREAD

**Discussion by RCNeil** [WordPress/gutenberg#43435] [Developer Experience]
I am attempting to set some default attributes for my `core/cover` block in my `theme.json` file.  I would like the min-height attribute to be 300px by default.  Here is my theme/json code:

`{
    "version": 2,
    "settings": {
		"layout": {
			"contentSize": "1080px",
			"wideSize": "1440px"
		},
		"spacing": {
			"blockGap" : true,
			"margin": true,
			"padding": true,
			"units": [ "rem", "px", "%"]
		},
		"blocks": {
			"core/cover": {
				"minHeight" : 300,
				"minHeightUnit": "px"
			}
		}
	}
}`

Is this attribute not able to be set?  It doesn't appear to be taking the value, though I am referring to the theme.json defaults in the block.json for that block https://github.com/WordPress/WordPress/blob/master/wp-includes/blocks/cover/block.json

Thanks for the help!

**Comment by t-hamano** [comment]
Yes. Currently in theme.json it is not possible to set default values for attributes defined in blocks.
If you use [blocks.registerBlockType filter](https://github.com/WordPress/gutenberg/blob/trunk/docs/reference-guides/filters/block-filters.md#blocksregisterblocktype) instead, you might be able to set a default value.

**Comment by cloudcreationsjess** [comment]
would love to be able to set attributes of blocks instead of overwriting this with css +1
